renderer/components/sample.tsx

```diff
@@ -1,16 +1,22 @@
-import React from 'react';
-import { Text, Box } from 'rebass';
+import React, { useRef, useEffect } from 'react';
+import { Text, Box, Flex } from 'rebass';
 
 import { Sound } from '../../@types';
 import { AudioPlayer } from './audio-player';
 import useTheme from '../hooks/useTheme';
+import useIpcService from '../hooks/useIpcService';
 import Stack from './stack';
 import getFileName from '../../util/getFileName';
 
 const Sample = ({ sound }: { sound: Sound }) => {
     const theme = useTheme();
+    const ipcService = useIpcService();
     const { filename: path } = sound;
     const filename = getFileName(path);
+    const onDrag = (e) => {
+        event.preventDefault();
```

Comment:
`event` should be undefined.
```suggestion
        e.preventDefault();
```

Reply:
@julesyoungberg Good find, weird that it did not show any errors.

Reply:
yeah, but that's javascript for ya 🤷